src/org/gk/render/RenderableProteinDrug.java

```diff
@@ -2,10 +2,10 @@
 
 import java.awt.Color;
 
-public class RenderableProteinDrug extends RenderableProtein {
+public class RenderableProteinDrug extends RenderableProtein implements DefaultRenderConstants {
```

Comment:
??

Reply:
Fixed in [e77f0a9](https://github.com/reactome/CuratorTool/pull/23/commits/e77f0a99bd4c6699021ff050f23d7c451e9e7d35).